app/src/main/res/layout/content_messages.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/content_messages"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior"
+        tools:showIn="@layout/fragment_messages"
+        tools:context="org.desarrolladorslp.teckersapp.ui.messages.MessageFragment" >
+
+    <ListView
```

Comment:
We don't usually use `ListView`, please use `RecyclerView`

Reply:
okey I will change it.